tests/test_result/test_result_equality.py

```diff
@@ -82,3 +84,27 @@ def test_immutability_success():
 
     with pytest.raises(AttributeError):
         Success(1).missing  # type: ignore # noqa: WPS428
+
+
+def test_success_immutable_copy():
+    """Ensures that Success returns it self when passed to copy function."""
+    success: Result = Success(1)
```

Comment:
Do we need `: Result` part here?

Reply:
Maybe don't, I'll try without this part!